get_subscription.py

```diff
@@ -87,12 +87,14 @@ def mount_get_root(mount):
 args = parser.parse_args()
 
 try:
-    appids = [app_to_sub[s] for s in args.appids]
+    subids = [app_to_sub[a] for a in args.appids if re.match('^\w+$', a)]
```

Comment:
It would be good to split this regexp out into a utility function in one of the libraries, both so that it can be reused elsewhere, and so that it has a method name which makes it a bit more obvious what it does.

Note that the regexp as it stands is equivalent to the character class `[a-zA-Z0-9_]`, which is not ‘all ASCII characters’ as Jasper suggested; but this is fine by me. We can always make it less restrictive later if needed.